README.md

```diff
@@ -1,40 +1,108 @@
 
+![BTCPay nopCommerce Banner](nopCommerceAcceptBitcoin.png)
 
 ## Plugin Overview
 
-This plugin allows you to easily integrate Bitcoin payments into your NopCommerce website using BTCPay Server — a free, self-hosted and open-source payment gateway solution designed to revolutionize Bitcoin payments. Our seamless integration with NopCommerce ensures a hassle-free connection to your self-hosted BTCPay Server. 
+This plugin allows you to easily start accepting Bitcoin payments in your nopCommerce e-commerce store using BTCPay Server — a free, self-hosted and open-source payment gateway. 
 
-Experience the simplicity of accepting Bitcoin payments with just a few straightforward steps. You can configure the plugin either automatically or manually, depending on your preferences and requirements.
+## Features
 
-## Automatic Configuration
+- 0% Fees: Enjoy a payment gateway with no fees. Yes, really!
+- No complicated KYC needed, you're your own self-hosted payment gateway
+- Automated invoice management and refunds
+- Simple and customizable checkout UX, available in 30+ languages
+- Direct payments to your wallet, no intermediary
+- Reporting and accounting – CSV exports
+- Real-time exchange price tracking for correct payment amounts
+- Community-driven support
+- Extensive documentation and video tutorials
 
-1. Enter Url to your BTCPay Server into "BTCPay Url" field. (e.g. https://mainnet.demo.btcpayserver.org)
-2. Click on the "Configure automatically" button to be redirected to the API authorization page on your BTCPay server
-3. On BTCPay authorization page: Select the store you want to connect to your NopCommerce (you might need to login first)
-4. Click on "Authorize App" button and you will be redirected back to your NopCommerce
-3. The "API Key", "BTCPay Store ID" and "Webhook Secret" fields will be automatically filled and a webhook created
-4. Click "Save" button at the top right to persist the configuration
-5. Congrats, the configuration is now done.
+## Getting Started 
 
-## Manual Configuration
+### Download and installation
 
-Ensure that the following fields are filled out: "BTCPay Url," "API Key," "BTCPay Store ID," and "WebHook Secret."
+The plugins can be downloaded through [nopCommerce Marketplace](https://www.nopcommerce.com/marketplace/) or [GitHub](https://github.com/btcpayserver/nopcommerce) 
 
-To create the BTCPay API key, [read this](https://docs.btcpayserver.org/VirtueMart/#22-create-an-api-key-and-configure-permissions).
-- Note: If you want to use the Refund feature, you must also add the "Create non-approved pull payments" permission. After a refund, an order note is created where you can copy the pull payments link and send to your customer (this order note is also visible by the customer). The customer can request the refund on that page.
+For installing the plugin through nopCommerce, you can either upload the plugin directly from nopCommerce `Configuration > Local plugins > Upload plugin or a theme`
```

Comment:
this sentence seems to be not finished, it says "either" but then there is no alternative way of install described, marketplace?

Reply:
Nice catch, fixed https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e